src/v2/view-builder/views/ov/EnrollAuthenticatorOVPollView.js

```diff
@@ -0,0 +1,118 @@
+import { View, loc } from 'okta';
+import BaseForm from '../../internals/BaseForm';
+import { isAndroid, isIOS } from '../../../../util/BrowserFeatures';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import AuthenticatorEnrollFooter from '../../components/AuthenticatorEnrollFooter';
+import Link from '../../components/Link';
+import { getSwitchOVEnrollChannelLink } from '../../utils/LinksUtil';
+import polling from '../shared/polling';
+import hbs from 'handlebars-inline-precompile';
+
+const Body = BaseForm.extend(Object.assign(
+  polling,
+  {
+    title () {
+      const selectedChannel = this.options.appState.get('currentAuthenticator').contextualData.selectedChannel;
+      let title;
+      switch (selectedChannel) {
+      case 'email':
+        title = loc('oie.enroll.okta_verify.setup.email.title', 'login');
+        break;
+      case 'sms':
+        title = loc('oie.enroll.okta_verify.setup.sms.title', 'login');
+        break;
+      default:
+        title = loc('oie.enroll.okta_verify.setup.title', 'login');
+      }
+      return title;
+    },
+    className: 'oie-enroll-ov-poll',
+    noButtonBar: true,
+    initialize () {
+      BaseForm.prototype.initialize.apply(this, arguments);
+      if ((isAndroid() || isIOS()) &
+        this.options.appState.get('currentAuthenticator').contextualData.selectedChannel === 'qrcode') {
+        this.options.appState.trigger('switchForm', 'enroll-with-another-channel');
+      }
+      this.listenTo(this.model, 'error', this.stopPolling);
+      this.startPolling();
+    },
+    getUISchema () {
+      const schema = [];
+      const contextualData = this.options.appState.get('currentAuthenticator').contextualData;
+      const selectedChannel = contextualData.selectedChannel;
+      if (selectedChannel === 'qrcode') {
```

Comment:
Why is it better to put this logic in getUISchema vs initialize? Is it because there may be some fields we do not want to render?

Reply:
if there is a schema and you want text before or after you can customize here or else initialize.